app/frontend/wallet/cardano-trezor-crypto-provider.js

```diff
@@ -14,6 +12,30 @@ const CardanoTrezorCryptoProvider = (ADALITE_CONFIG, walletState) => {
     throw new Error(`Unsupported derivation scheme: ${state.derivationScheme.type}`)
   }
 
+  const deriveXpub = generateDeriveXpubFn(async (absDerivationPath) => {
+    const response = await TrezorConnect.cardanoGetPublicKey({
+      path: absDerivationPath,
+      showOnTrezor: false,
+    })
+    if (response.error || !response.success) {
+      return Promise.reject(response || 'operation failed')
+    }
+    return Buffer.from(response.payload.publicKey, 'hex')
+  }, state.derivationScheme)
```

Comment:
I wonder whether we shouldn't switch the order of arguments. This line is quite confusing